sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -336,6 +359,25 @@ export class InstitutionController extends BaseController {
       );
     }
 
+    /** A legal signing authority role can be added to only one user per institution */
+    const addLegalSigningAuthorityExist = payload.permissions.some(
```

Comment:
i saw this logic is used multiple times, how about creating a common function for it and rescue it

Reply:
Resolving it as discussed.